packages/components/src/common/svgs/IconVirusCircleSLarge.svg

```diff
@@ -0,0 +1,20 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M20.7351 16.4951C20.7351 18.7733 18.8883 20.6202 16.6101 20.6202C14.3319 20.6202 12.4851 18.7733 12.4851 16.4951C12.4851 14.2169 14.3319 12.37 16.6101 12.37C18.8883 12.37 20.7351 14.2169 20.7351 16.4951ZM17.7286 18.6955C18.0487 18.4404 18.2087 18.0912 18.2087 17.6481C18.2087 17.3281 18.1236 17.0584 17.9536 16.8391C17.7835 16.6175 17.469 16.3971 17.0103 16.1778C16.6634 16.0122 16.4429 15.898 16.3489 15.8354C16.2572 15.7705 16.19 15.7045 16.1475 15.6373C16.105 15.5679 16.0837 15.4874 16.0837 15.3956C16.0837 15.2479 16.1363 15.1282 16.2415 15.0364C16.3467 14.9447 16.4978 14.8988 16.6947 14.8988C16.8603 14.8988 17.0282 14.9201 17.1982 14.9626C17.3706 15.0051 17.5876 15.0801 17.8495 15.1875L18.1852 14.3785C17.9323 14.2688 17.6895 14.1838 17.4567 14.1233C17.2262 14.0629 16.9834 14.0327 16.7283 14.0327C16.2091 14.0327 15.8018 14.1569 15.5063 14.4053C15.2132 14.6537 15.0666 14.995 15.0666 15.4292C15.0666 15.6597 15.1113 15.8611 15.2009 16.0334C15.2904 16.2058 15.4101 16.3568 15.5601 16.4866C15.71 16.6164 15.9349 16.7518 16.2348 16.8928C16.5548 17.045 16.7663 17.1558 16.8693 17.2251C16.9744 17.2945 17.0539 17.3673 17.1076 17.4433C17.1613 17.5172 17.1882 17.6022 17.1882 17.6985C17.1882 17.8708 17.1266 18.0006 17.0035 18.0879C16.8827 18.1752 16.7081 18.2188 16.4799 18.2188C16.2896 18.2188 16.0793 18.1886 15.8488 18.1282C15.6205 18.0677 15.3407 17.9637 15.0095 17.816V18.7828C15.4124 18.9797 15.8756 19.0782 16.3993 19.0782C16.9677 19.0782 17.4108 18.9506 17.7286 18.6955Z" stroke="#3867FA" stroke-width="0.1925" mask="url(#path-1-outside-1_1810_444)"/>
+<path d="M11.6392 15.1203C11.8171 14.4757 12.117 13.8818 12.5126 13.3649C12.3562 12.6756 11.7397 12.1609 11.003 12.1609C10.1482 12.1609 9.45516 12.8539 9.45516 13.7088C9.45516 14.5637 10.1482 15.2567 11.003 15.2567C11.2298 15.2567 11.4452 15.2079 11.6392 15.1203Z" stroke="#3867FA" stroke-width="0.1925" mask="url(#path-1-outside-1_1810_444)"/>
```

Comment:
These stroke paths should be removed. If you modify the icon's color, the stroke color remains unchanged, which is not what we expect. In my opinion, the ideal approach would be to fix the SVG file itself and unite the stroke paths to the main object paths.  

<img width="1324" alt="Screenshot 2023-07-14 at 1 46 12 PM" src="https://github.com/chanzuckerberg/sci-components/assets/927990/61b72747-87a7-4b5a-9235-072df7cd2a21">

Reply:
@masoudmanson Thanks, good catch, I should have tried looking at it in `warning` color, since it's easier to see this kind fo thing I am realizing!
When you say
> fix the SVG file itself and unite the stroke paths to the main object paths

do you mean I can do this in he code? Or that is something to do in the design, before exporting (I know there is a way to unite pieces in Figma, for example).

If the former, is there a quick way to explain how? I have not worked with SVG code very much yet. Thanks!

Reply:
I meant to fix the SVG in Figma then export it :D

Reply:
Okay thanks! @clarsen-czi can you update the icon like Masoud is suggesting, if that makes sense when you get a chance? Thansk!!

Reply:
@liaprins I'll work on getting it fixed in Figma and send over an updated version. I'll need to do some troubleshooting to figure it out because the standard version of Virus icon without the "s" also has that thin stroke on it, but for whatever reason the stroke isn't causing the same issue 🤔

Reply:
> @liaprins I'll work on getting it fixed in Figma and send over an updated version. I'll need to do some troubleshooting to figure it out because the standard version of Virus icon without the "s" also has that thin stroke on it in Figma, but for whatever reason the stroke hasn't caused the same issue in code 🤔

Hmm that made me think it still could be a code-related issue, so I actually did some more exploring in the code and found the issue! So I'll push another commit soon and have you all take another look!

Reply:
@clarsen-czi While working on the Icon bank, I fixed the old icons. There were a few icons with this issue, but it was so long ago that I can't remember which ones had the stroke!  😥